content/doc/developer/publishing/wiki-page.adoc

```diff
@@ -73,10 +73,10 @@ There are the following migration steps:
 
 === Exporting Wiki documentation (automatic way)
 
-The link:https://jenkins-wiki-exporter.g4v.dev/[Jenkins Wiki Exporter] can export plugin documentation to Markdown or Asciidoc.
+The link:https://jenkins-wiki-exporter.jenkins.io/[Jenkins Wiki Exporter] can export plugin documentation to Markdown or Asciidoc.
 
-. Go to the link:https://jenkins-wiki-exporter.g4v.dev/[Jenkins Wiki Exporter] service
-. Specify the plugin ID you want to export in the field
+. Go to the link:https://jenkins-wiki-exporter.jenkins.io/[Jenkins Wiki Exporter] service
+. Specify the plugin Wiki UR or ID in the field
```

Comment:
I was able to use the URL, so I assume the abbreviation here is intended for universal resource locator or possibly for universal resource identifier.  UR will probably be understood by readers, but I'm more accustomed to URL.
```suggestion
. Specify the plugin Wiki URL or ID in the field
```